hypha/apply/review/views.py

```diff
@@ -51,8 +51,16 @@ def get_context_data(self, **kwargs):
         )
 
 
-def get_fields_for_stage(submission):
+def get_fields_for_stage(submission, user=None):
     forms = submission.get_from_parent('review_forms').all()
+    external_review_forms = submission.get_from_parent('external_review_forms').all()
+
+    # Use ExternalReviewForm if submission's stage has external review and external review form is attached to fund.
+    # ExternalReviewForm is only for non-staff reviewers(external reviewers)
+    if submission.stage.has_external_review and external_review_forms:
+        if user and not user.is_apply_staff:
```

Comment:
External forms should only be rendered for reviewers. So instead of checking a user not staff we could check if a user is reviewer..?